packages/react-sdk/src/core/hooks/useObserveDevices.ts

```diff
@@ -0,0 +1,137 @@
+import { useEffect, useState } from 'react';
+import { Observable, pairwise } from 'rxjs';
+import {
+  getAudioDevices,
+  getAudioOutputDevices,
+  getVideoDevices,
+} from '@stream-io/video-client';
+
+/**
+ * Observes changes in connected devices and maintains an up-to-date array of connected MediaDeviceInfo objects.
+ * @param observeDevices
+ */
```

Comment:
I think the `@category Device Management` lines are missing, so the new hooks are under "Device Management" category in the docs: https://staging.getstream.io/video/docs/react/call-engine/hooks-and-contexts/#device-management